app/build.gradle

```diff
@@ -147,6 +148,7 @@ dependencies {
     testCompile libraries.equalsVerifier
     testCompile libraries.okHttpMockWebServer
     testCompile libraries.mockitoCore
+    testCompile libraries.devMetricsNoOp
```

Comment:
Not sure I understand how this works now (will check plugin sources later), since we don't specify dependency on real version manually, does AndroidDevMetrics plugin excludes dependency from test? @frogermcs